types/index.ts

```diff
@@ -0,0 +1,28 @@
+import WS from "ws";
+
+export type UnixDate = number;
+
+export namespace Keva {
+  // TODO: Make this more complete
+  export interface Transaction {
+    tx_hash: string;
+    // FIXME: This is probably the wrong type for `n`
+    n: string;
```

Comment:
The value of `n` looks like this:
```
n: [ 'Nfw2WYkGoSKve74cCfEum67x8bFgpHygxg', 1 ]
```
It is an array type, the first element is the namespace Id, while the second element is the position of the transaction (which contains the namespace) in the block.

So indeed it is not a string.